docs/usage.md

```diff
@@ -189,6 +192,12 @@ The full path to GTF file can be specified for annotating peaks. Note that the G
 --gtf '[path to GTF file]'
 ```
 
+### `--bed`
```

Comment:
We should be able to automatically generate this file from the GTF if not supplied. This is what we do in the rnaseq pipeline: https://github.com/nf-core/rnaseq/blob/44f1525d7adfa3a6afcafdcf7730eca78a1a2511/main.nf#L433-L453